.github/workflows/integration.yml

```diff
@@ -0,0 +1,41 @@
+name: HighFive_Integration_tests
+
+on:
+  push:
+    branches:
+      - ci_test
+      - release/**
+  pull_request:
+    branches:
+      - master
+      - release/**
+    paths-ignore:
+      - '**.md'
+      - '**.rst'
+      - 'doc/**'
+
+env:
+  BUILD_TYPE: RelWithDebInfo
+  INSTALL_DIR: ${{github.workspace}}/install
+
+jobs:
+
+  Linux_Compilers:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: "Install libraries"
+      run: sudo apt-get update && sudo apt-get install libboost-all-dev libhdf5-dev libeigen3-dev
+
+    - name: Build
+      run: source $GITHUB_WORKSPACE/.github/build.sh
```

Comment:
This does not import anything from the child environment… so `sh` would seem a better fit.

Reply:
for the other way around, we want to be free to use parent variables in the child environment, exported or not.